xml/xml.xsd

```diff
@@ -0,0 +1,160 @@
+<?xml version="1.0"?>
```

Comment:
This change should use the more recent `xml.xsd` from https://raw.githubusercontent.com/VDVde/OJP/changes_for_v1.1/siri/xml/xml.xsd instead.

Reply:
This is fixed. Can you give your blessing?